compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIRGen.java

```diff
@@ -280,6 +279,7 @@ private BIRGen(CompilerContext context) {
     }
 
     public BLangPackage genBIR(BLangPackage astPkg) {
+        System.out.println("Writing :" + astPkg + "env " + symTable.typeEnv().envStats());
```

Comment:
Is it correct to use `System.out` to log the error?

Reply:
This PR is just to diagnose OOM issue in our CI, not intend to merge